lib/open_project/backlogs/hooks.rb

```diff
@@ -103,9 +103,13 @@ def view_versions_show_bottom(context = {})
       end
     end
 
-    def view_my_account(context = {})
-      context[:controller].send(:render_to_string,           partial: 'shared/view_my_account',
-                                                             locals: { user: context[:user], color: context[:user].backlogs_preference(:task_color), versions_default_fold_state: context[:user].backlogs_preference(:versions_default_fold_state) })
+    def view_my_settings(context = {})
+      context[:controller].send(
+          :render_to_string,
+          partial: 'shared/view_my_settings',
+          locals: { user: context[:user],
+            color: context[:user].backlogs_preference(:task_color),
+            versions_default_fold_state: context[:user].backlogs_preference(:versions_default_fold_state) })
```

Comment:
Line is too long. [108/100]<br>Align the elements of a hash literal if they span more than one line.